ttnn/cpp/ttnn/operations/pool/avgpool/avg_pool_pybind.hpp

```diff
@@ -59,8 +59,27 @@ void bind_global_avg_pool2d(py::module& module) {
 
 }  // namespace detail
 
-void py_module(py::module& module) { detail::bind_global_avg_pool2d(module); }
+void py_module(py::module& module) {
+    detail::bind_global_avg_pool2d(module);
+    module.def(
+        "average_pool_2d",
+        &average_pool_2d,
+        py::arg().noconvert(),
+        py::kw_only(),
+        py::arg("memory_config").noconvert() = operation::DEFAULT_OUTPUT_MEMORY_CONFIG,
+        py::arg("dtype").noconvert() = std::nullopt,
+        R"doc(
+        Average Pool 2D
+        It operates on tensors whose that have channels as the last dimension
```

Comment:
Review this sentence please
```suggestion
        It operates on tensors whose that have channels as the last dimension
```